### DISCUSSION TITLE
Import boards from Trello JSON export

### DISCUSSION THREAD

**Discussion by obbardc** [JordanKnott/taskcafe#67] [Ideas]
Trello offers a JSON export, it would be nice to be able to import cards from this format.

Backup to this format could be discussed later ;-)

**Comment by JordanKnott** [comment]
The ability to import projects is a feature that is planned! However it likely won't be implemented soon as I have many other "core" functionality to implement first

**Comment by Cupara** [comment]
It will be great when Trello import is implemented. I have 2 projects I want to transition to TaskCafe. I'm moving my 3rd one right now but I'm moving it manually so no biggie.
